snforge_std/src/trace.cairo

```diff
@@ -2,48 +2,55 @@ use core::result::ResultTrait;
 use core::starknet::testing::cheatcode;
 use core::starknet::ContractAddress;
 
+/// Tree-like structure which contains all of the calls and sub-calls along with the results
 #[derive(Drop, Serde, PartialEq)]
 struct CallTrace {
-    entry_point: CallEntryPoint,
-    nested_calls: Array<CallTrace>,
+    entry_point: CallEntryPoint, /// The topmost entry point of this trace
```

Comment:
```suggestion
    /// A summary of a call to a contract entry point
    entry_point: CallEntryPoint,
```

Reply:
I don't see a point in copy-pasting the struct doc in the doc to the field, it doesn't bring any extra information to the table. I'd rather just remove it instead

Reply:
Sound good